### DISCUSSION TITLE
How to implement nnUNet for BraTs2020 baseline?

### DISCUSSION THREAD

**Discussion by mehnaz1985** [MIC-DKFZ/nnUNet#1174] [Q&A]
Could you please enlighten me about the dataset conversion file of task082_BraTs2020? I want to implement the nnUNet baseline for segmentation first and then will try to modify. for medical decathlon there are simple commands for dataset conversion, is there any command for brats 2020 data conversion? Thanks for your time and support.

**Comment by FabianIsensee** [comment]
did you read the dataset conversion file? It;s all in there:
https://github.com/MIC-DKFZ/nnUNet/blob/aa53b3b87130ad78f0a28e6169a83215d708d659/nnunet/dataset_conversion/Task082_BraTS_2020.py#L621
